hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetBulkUpdateWindowLayout.java

```diff
@@ -336,9 +337,15 @@ public void onUploadCompletion() {
         minimizeButton.setEnabled(false);
     }
 
-    private static String getFormattedCountLabelValue(final int succussfulUploadCount, final int failedUploadCount) {
-        return new StringBuilder().append("Successful :").append(succussfulUploadCount)
-                .append("<font color=RED> Failed :").append(failedUploadCount).append("</font>").toString();
+    private String getFormattedCountLabelValue(final int succussfulUploadCount, final int failedUploadCount) {
```

Comment:
typos in _succussfulUploadCount_ parameter, and I would rename the _ stringBuilder_ local variable to e.g. _countLabelBuilder_